Assets/SEE/UI/Window/PropertyWindow/PropertyWindow.cs

```diff
@@ -0,0 +1,845 @@
+                }
+                /// String Attributes
+                if (GraphElement.StringAttributes.Count > 0 && contextMenu.Filter.IncludeStringAttributes)
+                {
+                    DisplayGroup("String Attributes", GraphElement.StringAttributes);
+                }
+
+                /// Int Attributes
+                if (GraphElement.IntAttributes.Count > 0 && contextMenu.Filter.IncludeIntAttributes)
+                {
+                    DisplayGroup("Int Attributes", GraphElement.IntAttributes);
+                }
+
+                /// Float Attributes
+                if (GraphElement.FloatAttributes.Count > 0 && contextMenu.Filter.IncludeFloatAttributes)
+                {
+                    DisplayGroup("Float Attributes", GraphElement.FloatAttributes);
+                }
+            }
+
+            /// Creates the items for the name type group.
+            void GroupByNameType()
+            {
+                if (GraphElement.ToggleAttributes.Count > 0 && contextMenu.Filter.IncludeToggleAttributes)
+                {
+                    Dictionary<string, bool> toggleDict = GraphElement.ToggleAttributes.ToDictionary(item => item, item => true);
+                    if (groupHolder.ContainsKey("Header"))
+                    {
+                        DisplayAttributes(toggleDict, group: "Header");
+                    }
+                    else
+                    {
+                        Dictionary<string, (string, GameObject gameObject)> toggleItems = DisplayAttributes(toggleDict);
+                        groupHolder.Add("Header", toggleItems.Values.Select(x => x.gameObject).ToList());
+                        expandedItems.Add("Header");
+                    }
+                }
+                Dictionary<string, object> attributes = new();
+                if (GraphElement.StringAttributes.Count > 0 & contextMenu.Filter.IncludeStringAttributes)
+                {
+                    foreach (KeyValuePair<string, string> pair in GraphElement.StringAttributes)
+                    {
+                        attributes.Add(InsertDotInFirstPascalCase(pair.Key), pair.Value);
+                    }
+                }
+                if (GraphElement.IntAttributes.Count > 0 & contextMenu.Filter.IncludeIntAttributes)
+                {
+                    foreach (KeyValuePair<string, int> pair in GraphElement.IntAttributes)
+                    {
+                        string key = pair.Key;
+                        /// Block for old gxl files.
+                        if (key.Contains("SelectionRange") && !key.Contains("Source"))
+                        {
+                            key = "Source." + key;
+                        }
+                        key = InsertDotInFirstPascalCase(pair.Key);
+                        /// To remove duplicates it is needed to remove the old one. <see cref="GraphWriter"/> L.313.
```

Comment:
The line number will likely become outdated at some point. If possible, reference that part of GraphWriter by using the method name, or some other descriptor.